tests/phpunit/elementor/core/upgrade/test-upgrades.php

```diff
@@ -567,6 +567,16 @@ public function test_v_3_4_8_fix_font_awesome_default_value_from_1_to_yes() {
 		$this->assertEquals( 'yes', get_option( Icons_Manager::LOAD_FA4_SHIM_OPTION_KEY ) );
 	}
 
+	public function test_v_3_6_0_remove_old_elements_usage() {
+		$this->ensure_removed_old_usage_data(
```

Comment:
Why it's a separated method?

Reply:
In case there will be more option naming changes....
Do you want it, in this test only?

Reply:
yes, in this test only

Reply:
Handled.